docs/rules/no-unsafe-negation.md

```diff
@@ -72,6 +70,42 @@ if (!(foo) in object) {
 }
 ```
 
+## Options
+
+This rule has an object option:
+
+- `"enforceForOrderingRelations": false` (default) allows negation of the left-hand side of ordering relational operators (`<`, `>`, `<=`, `>=`)
+- `"enforceForOrderingRelations": true` disallows negation of the left-hand side of ordering relational operators
+
+### enforceForOrderingRelations
+
+The enforceForOrderingRelations option extends the rule to apply to all [Relational Operators](https://www.ecma-international.org/ecma-262/10.0/index.html#sec-relational-operators), not just `in` and `instanceof`.
+
+With this option set to true, the rule is enforced for all of the following:
+
+- `in` operator.
+- `instanceof` operator.
+- `<` operator.
+- `>` operator.
+- `<=` operator.
+- `>=` operator.
```

Comment:
I think it's better here to just say what's the option additionally enforcing, without mentioning 'all relational operators', `in` and `instanceof`. The rule might have more enhancements in the future, it's better to be explicit.